src/tdd/Receipt.java

```diff
@@ -0,0 +1,31 @@
+package tdd;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+public class Receipt {
+  private static Receipt receipt = new Receipt();
+
+  public static Receipt getReceiptInstance(){
+    return receipt;
+  }
+
+  public void printReceipt(ArrayList<IProduct> orders, BigDecimal totalTaxes, BigDecimal purchaseTotal){
+    System.out.println("Cart Item               Price");
+    System.out.println("-------------------------------\n");
+    processPurchases(orders);
+    System.out.println("Sales Taxes: $" + totalTaxes + "\n");
+    System.out.println("Total: $" + purchaseTotal + "\n");
+    System.out.println("-------------------------------\n");
+    System.out.println();
+  }
+
+  public String processPurchases(ArrayList<IProduct> orders) {
```

Comment:
Should have been a toString() method that would be used both in printReceipt and in UT.